### ISSUE TITLE
Proposed rule: match zip() arguments and unpacked variables

### ISSUE THREAD

**Issue by jenstroeger** [PyCQA/flake8-bugbear#545]
In an older code blob of mine I came across this oddity:
```python
for a, b, _ in zip(iter_a, iter_b, iter_c):
    ...
```
which should be simplified to
```python
for a, b in zip(iter_a, iter_b):
    ...
```
by removing `iter_c` entirely. Turns out I had this pattern in two other places as well 🤦🏻‍♂️

Would this not be a useful idiom to look for and warn about? (Assuming `_` isn’t used in the body of the loop…)

**Comment by cooperlees** [comment]
Yeah, I feel this is a unneeded code smell. So I'll accept a new lint here recommending just removing from the `zip()` rather than ignoring the element. Please add tests like all other lints and to README.

I think using "`_`" is a thing all devs should not do ... so this helps that ...
